File: Counting Duplicates/counting-duplicates.py
def duplicate_count(text):
    """
    :type text: str
    :rtype: int
    """
    text = text.lower()
    return len([x for x in set(text) if text.count(x) > 1])


print(duplicate_count("abcde"))  # 0
print(duplicate_count("aabbcde"))  # 2
print(duplicate_count("Indivisibilities"))  # 2

File: Counting Duplicates/counting-duplicates.js
/**
 * Duplicate count function
 *
 * @return {number}
 * @param {string} text
 */
function duplicateCount(text) {
  text = text.toLowerCase();

  let res = 0;
  new Set(text).forEach(function (value) {
    if (text.split(value).length - 1 > 1) {
      res++;
    }
  });

  return res;
}

console.log(duplicateCount('abcde')); // 0
console.log(duplicateCount('aabbcde')); // 2
console.log(duplicateCount('Indivisibilities')); // 2